### PR TITLE
postcss-custom-media: avoid complex generated CSS in more cases

### PR DESCRIPTION
**Pull request by romainmenke**
fixes : https://github.com/csstools/postcss-plugins/issues/705

### CODE REVIEW

File: plugins/postcss-custom-media/src/transform-at-media/transform-at-media.ts [hunk 1]
```diff
@@ -1,5 +1,5 @@
 import { alwaysTrue, neverTrue } from './always-true-or-false';
-import { isGeneralEnclosed, isMediaAnd, isMediaConditionList, isMediaFeature, isMediaFeatureBoolean, isMediaNot, isMediaOr, isMediaQueryInvalid, isMediaQueryWithType, MediaQuery, newMediaFeaturePlain, parse } from '@csstools/media-query-list-parser';
+import { isGeneralEnclosed, isMediaAnd, isMediaConditionList, isMediaFeature, isMediaFeatureBoolean, isMediaNot, isMediaOr, isMediaQueryInvalid, isMediaQueryWithType, MediaFeature, MediaQuery, newMediaFeaturePlain, parse } from '@csstools/media-query-list-parser';
```
Comment (Antonio-Laguna) [inline plugins/postcss-custom-media/src/transform-at-media/transform-at-media.ts:2]: [nit] Should we make this a line per import so it's easier to see/maintain?

File: plugins/postcss-custom-media/src/transform-at-media/transform-at-media.ts [hunk 2]
```diff
@@ -104,6 +104,29 @@ export function transformComplexMediaQuery(mediaQuery: MediaQuery, replacements:
 
 		const replacement = replacements.get(name);
 		if (replacement) {
+
+			if (replacement.truthy.length === 1 && mediaQueryIsSimple(replacement.truthy[0])) {
+				let mediaFeature: MediaFeature | null = null;
+				const replacementMediaQuery = replacement.truthy[0];
```
Comment (Antonio-Laguna) [inline plugins/postcss-custom-media/src/transform-at-media/transform-at-media.ts:110]: [nit] Should this use destructuring?

**Review by Antonio-Laguna — APPROVED** [review]
This does look great to me!

Left really minor comments but feel free to discard them if you don't really agree

File: plugins/postcss-custom-media/src/transform-at-media/transform-at-media.ts (hunk 1 above)
Comment (romainmenke) [reply plugins/postcss-custom-media/src/transform-at-media/transform-at-media.ts]: Good catch!

The auto import feature in typescript makes me lazy.
It just keeps adding things to the list and I never really look at it :D

File: plugins/postcss-custom-media/src/transform-at-media/transform-at-media.ts (hunk 2 above)
Comment (romainmenke) [reply plugins/postcss-custom-media/src/transform-at-media/transform-at-media.ts:124]: I liked the parallel with the condition : `(replacement.truthy.length === 1 && mediaQueryIsSimple(replacement.truthy[0]))`

**Pull request merged**